t_http2/test_h2_sticky_scheduler.py

```diff
@@ -0,0 +1,164 @@
+                    cookie enforce;
+                    secret "f00)9eR59*_/22";
+                }
+            }
+            tls_match_any_server_name;
+            tls_certificate ${tempesta_workdir}/tempesta.crt;
+            tls_certificate_key ${tempesta_workdir}/tempesta.key;
+            cache 0;
+            cache_fulfill * *;
+            block_action attack reply;
+            http_chain {
+                host == "bad.com"	-> block;
+                host == "good.com" -> v_good;
+            }
+        """,
+    }
+
+    backends = [
+        {
+            'id': 'nginx',
+            'type': 'nginx',
+            'port': '8000',
+            'status_uri': 'http://${server_ip}:8000/nginx_status',
+            'config': """
+                pid ${pid};
+                worker_processes  auto;
+                events {
+                    worker_connections   1024;
+                    use epoll;
+                }
+                http {
+                    keepalive_timeout ${server_keepalive_timeout};
+                    keepalive_requests ${server_keepalive_requests};
+                    sendfile         on;
+                    tcp_nopush       on;
+                    tcp_nodelay      on;
+                    open_file_cache max=1000;
+                    open_file_cache_valid 30s;
+                    open_file_cache_min_uses 2;
+                    open_file_cache_errors off;
+                    error_log /dev/null emerg;
+                    access_log off;
+                    server {
+                        listen        ${server_ip}:8000;
+                        location / {
+                            return 200;
+                        }
+                        location /nginx_status {
+                            stub_status on;
+                        }
+                    }
+                }
+            """,
+        },
+    ]
+
+    def test_h2_cookie_scheduler(self):
+        """
+        Test for sticky cookie scheduler by issue.
+
+        Cookies are taken from response.
+
+        1. client request URI with Host http://good.com/
+        2. tempesta responds with 302 and sticky cookie
+        3. client repeat the request with sticky cookie set
+        4. tempesta forwards the request to server and
+        forwards its response to the client with 200
+        5. client requests URI with Host http://bad.com/ with sticky cookie set
+        6. tempesta filtering out request
+
+        """
+        klog = dmesg.DmesgFinder()
+        curl_init = self.get_client('curl-init')
+
+        self.start_all_servers()
+        self.start_tempesta()
+
+        # perform `init` request
+        curl_init.start()
+        self.wait_while_busy(curl_init)
+        resp_init = curl_init.resq.get(True, 1)[0]
+        resp_init = parse_response(resp_init, encoding='latin-1')
+        set_cookie = resp_init.headers.get('set-cookie')
+        self.assertEqual(
+            int(resp_init.status),
```

Comment:
`status` has already been converted to `int` in `helpers.response_parser.Response.__init__`.

Reply:
I removed type casting from class, but I see no problem in double type casting